utils/constants.py

```diff
@@ -0,0 +1,14 @@
+PROJECTS_ABBREV = [
```

Comment:
This is the same as lib/__init__.py , except for line 7, ALL... are both files needed?